### PR TITLE
chore(frontend): Generate TypeScript client for Kubernetes_executor_config

### PR DESCRIPTION
**Pull request by jlyaoyuli**
To implement the PlatformSpec-related features we will need typescript client of `kubernetes_executor_config.proto`.

Add a new section **Platform Spec** in the README file to describe the usage of new-added script (`build:platform-spec:kubernetes-platform`)

### CODE REVIEW

File: frontend/scripts/k8splatformspec.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -ex
+
+# Run the following command under /frontend to execute this file
+# npm run build:kubernetes-platform-spec
+
+
+# Convert buffer to runtime object using protoc
+# Download google protos as dependencies.
```
Comment (chensun) [inline frontend/scripts/k8splatformspec.sh:10]: nit: there's no download in this script.

**Review by chensun — APPROVED** [review]
/lgtm
/approve

File: frontend/package.json [hunk 2]
```diff
@@ -68,6 +68,7 @@
     "build": "npm run lint && EXTEND_ESLINT=true react-scripts build",
     "build:protos": "cd .. && node frontend/scripts/gen_grpc_web_protos.js",
     "build:replace": "node scripts/replace_protos.js",
+    "build:k8s-platform-spec": "bash ./scripts/k8splatformspec.sh",
```
Comment (zijianjoy) [inline frontend/package.json:71]: Can we add documentation about how to use this script?

Comment (zijianjoy) [inline frontend/package.json:71]: I would suggest to cascade the hierarchy here: `build:platform-spec:kubernetes-platform`

**Comment by zijianjoy** [comment]
/lgtm
/approve

Thank you Joe!

**Pull request merged**